### PR TITLE
Upgrade falco to 0.31.1

### PR DESCRIPTION
**Pull request by NissesSenap**
Upgrade falco to 0.31.1
Upgrade falco-exporter to 0.8.0

### CODE REVIEW

**Comment by NissesSenap** [comment]
Or I just suck at writing the config rule... :D

**Comment by NissesSenap** [comment]
So without my rule it's not a problem to patch to the latest version in Azure.
I will verify in AWS as well.

**Comment by NissesSenap** [comment]
It works fine in AWS as well. For now we will just patch to the latest version.

File: CHANGELOG.md [hunk 1]
```diff
@@ -13,6 +13,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 ### Fix
 
 - [#614](https://github.com/XenitAB/terraform-modules/pull/620) Fix broken cluster autoscaler
+- [#617](https://github.com/XenitAB/terraform-modules/pull/617) Upgrade falco to 0.31.1
```
Comment (jimgus) [inline CHANGELOG.md:16]: Maybe file under "change" instead of "fix"?

**Review by jimgus — APPROVED** [review]

**Pull request merged**